src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs

```diff
@@ -454,7 +454,7 @@ private void ExecuteCompilationActionsCore(ImmutableArray<CompilationAnalyzerAct
                 analyzerStateOpt?.ProcessedActions.Add(operationAction);
```

Comment:
At some point in future, we should probably abstract out the common parts of code between:
 1. syntax node execution and operation execution
 2. code block execution and operation block execution.

I feel a bit worried that a bug fix in one execution path will miss its corresponding counterpart.